packages/react-instantsearch-dom/src/widgets/Answers.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState, useEffect, useMemo, useContext } from 'react';
+import { instantSearchContext } from 'react-instantsearch-core';
+import { createConcurrentSafePromise } from '../lib/createConcurrentSafePromise';
+import { debounce } from '../lib/debounce';
+
+/* eslint-disable react/prop-types */
+function DefaultAnswersComponent({ isLoading, hits = [] }) {
+  return isLoading ? (
+    <p>loading...</p>
+  ) : (
+    hits && hits.length > 0 && <pre>{JSON.stringify(hits, null, 2)}</pre>
```

Comment:
Doesn't it throw an error in case `(!hits || !hits.length) && !isLoading` ?

Reply:
If it's not loading (!isLoading) and there is no hits, then it will not render anything.
![image](https://user-images.githubusercontent.com/499898/102070310-3cbe6e80-3dff-11eb-9f41-6e5631c86a86.png)

Reply:
Oh yeah make sense!